src/sql/workbench/api/common/extHostModelView.ts

```diff
@@ -1384,6 +1384,12 @@ class TableComponentWrapper extends ComponentWrapper implements azdata.TableComp
 		this.setProperty('updateCells', v);
 	}
 
+	public appendData(v: any[][]): void {
```

Comment:
I think this better fits by using the $doAction method to do the action on the component https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/api/browser/mainThreadModelView.ts#L107

See https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/browser/modelComponents/tabbedPanel.component.ts#L133 for an example of this